pkg/endpoint/endpoint.go

```diff
@@ -411,9 +414,12 @@ func NewEndpointWithState(ID uint16, state string) *Endpoint {
 		state:         state,
 		hasBPFProgram: make(chan struct{}, 0),
 		controllers:   controller.NewManager(),
+		EventQueue:    eventqueue.NewEventQueueBuffered(25),
```

Comment:
Two nits here:
* Where does 25 come from?
* This declaration with magic number is now hardcoded across the three (sigh) endpoint initialization functions, at least the number should be a constant that can be easily tweaked (which also provides a convienient spot to place a comment about why the number 25 is an endpoint's favorite number and how everyone should love the number 25 too)

Reply:
I'll factor it out into a constant. Perhaps we should make it configurable as well for larger-scale clusters with lots of events.

Reply:
yeah, personally I don't have a very solid understanding of what reasonable buffer sizes for these things are. Support for custom values would go a long way there.

Seems like it's all well and good to queue up 25 events but if they're all full regeneration events then we're not going to be able to handle those very quickly with one goroutine, so this is really not pushing back enough on the callers... at the same time, if it's 25 policy revision bumps then it seems quite reasonable to handle lots of events in quick succession.

Reply:
(Similarly the policy package usage of EventQueue has magic numbers and I don't know where those buffer sizes come from either)

Reply:
I elected to keep the defaults the same for now - as we test in high-scale environments, we can figure out what is best for certain environments and change the defaults if needed.